### ISSUE TITLE
Setting a transit vehicle as start point

### ISSUE THREAD

**Issue by rosecitytransit** [conveyal/trimet-mod-otp#164]
Not sure if this is the right place for outsiders to bring up issues, but I was thinking that if you're adjusting trips based upon real-time transit info, how about making it so you can set the transit vehicle you're on as a start point?  Let's say you make it onto the first transit leg of a trip, but then a delay happens en route (or you decide to change your plans).  It would be good to use the predicted arrivals of the vehicle to calculate transfers.

**Comment by msrichardson** [comment]
Good idea.  It is more complex than it sounds, but we have it on our list of potential future enhancements.
